apps/src/p5lab/spritelab/defaultSprites.json

```diff
@@ -58,651 +58,651 @@
       "frameSize": {
-        "x": 121,
-        "y": 191
+        "x": 152,
+        "y": 193
       },
       "frameCount": 1,
       "looping": true,
-      "frameDelay": 12,
-      "version": "r.VceVMCHQHcfAm0rusww1GhlSW8q7LI"
+      "frameDelay": 2,
+      "version": "kBiszeGACcLTGTrqmS4laPVQKPGQnDln"
     },
     "c71f8624-bc69-4131-8af4-3db6b34b480f": {
       "name": "corgi",
-      "sourceUrl": "https://studio.code.org/v3/animations/pPIy6SojmqKru9-me0lakrIb69iO-IfiboxR51iuq8c/c71f8624-bc69-4131-8af4-3db6b34b480f.png?version=_b__M_fHWCHgaEO3qjARWGJXievl_570",
+      "sourceUrl": "https://studio.code.org/api/v1/animation-library/spritelab/VtKpUDKH2C9JjfbimUkhbbH.dHQ5t7ex/category_animals/corgi.png",
       "frameSize": {
-        "x": 541,
+        "x": 542,
         "y": 500
       },
       "frameCount": 1,
       "looping": true,
-      "frameDelay": 12,
-      "version": "_b__M_fHWCHgaEO3qjARWGJXievl_570"
+      "frameDelay": 2,
+      "version": "VtKpUDKH2C9JjfbimUkhbbH.dHQ5t7ex"
     },
     "1b3549d0-607f-4702-8fda-c9a2c2b951e4": {
       "name": "cow",
-      "sourceUrl": "https://studio.code.org/api/v1/animation-library/N3OcPnyUKH2ZmtAEFB7h7C9mZVJlfEWy/category_animals/cow.png",
+      "sourceUrl": "https://studio.code.org/api/v1/animation-library/spritelab/ROqEBwbLmX6qg9Acz.DUkfyl2gaMh4lP/category_animals/cow.png",
       "frameSize": {
         "x": 265,
         "y": 300
       },
       "frameCount": 1,
       "looping": true,
       "frameDelay": 2,
-      "version": "N3OcPnyUKH2ZmtAEFB7h7C9mZVJlfEWy"
+      "version": "ROqEBwbLmX6qg9Acz.DUkfyl2gaMh4lP"
     },
     "f72dadb4-f85a-4867-bae9-204290b806f4": {
       "name": "crab",
-      "sourceUrl": "https://studio.code.org/v3/animations/pPIy6SojmqKru9-me0lakrIb69iO-IfiboxR51iuq8c/f72dadb4-f85a-4867-bae9-204290b806f4.png?version=TUP.GCDJQyPspPAim1D_FCYvnVOg8hZz",
+      "sourceUrl": "https://studio.code.org/api/v1/animation-library/spritelab/SdV.w3j8_4Djmt1L5C_gEVpS2VTh53Od/category_animals/crab.png",
       "frameSize": {
         "x": 288,
         "y": 191
       },
       "frameCount": 1,
       "looping": true,
-      "frameDelay": 12,
-      "version": "TUP.GCDJQyPspPAim1D_FCYvnVOg8hZz"
+      "frameDelay": 2,
+      "version": "SdV.w3j8_4Djmt1L5C_gEVpS2VTh53Od"
     },
     "e4b42ce0-9510-46c2-910c-0c4b69cf612c": {
       "name": "elephant",
-      "sourceUrl": "https://studio.code.org/v3/animations/pPIy6SojmqKru9-me0lakrIb69iO-IfiboxR51iuq8c/e4b42ce0-9510-46c2-910c-0c4b69cf612c.png?version=ofSrM4hsFsE7Q_k3D7c8Ds7.JOZvkIBn",
+      "sourceUrl": "https://studio.code.org/api/v1/animation-library/spritelab/jK3q6UNbOMNbaiWBwclkhL3D8isE1cLQ/category_animals/elephant2.png",
       "frameSize": {
         "x": 99,
         "y": 97
       },
       "frameCount": 1,
       "looping": true,
-      "frameDelay": 12,
-      "version": "ofSrM4hsFsE7Q_k3D7c8Ds7.JOZvkIBn"
+      "frameDelay": 2,
+      "version": "jK3q6UNbOMNbaiWBwclkhL3D8isE1cLQ.JOZvkIBn"
```

Comment:
Looks like this version is different than the URL

Reply:
thanks for catching these :)